tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -124,53 +124,113 @@ def get_virtctl():
-            # Create or verify existing SC in all clusters
             existing_sc_list = get_all_storageclass()
             if sc_name in existing_sc_list:
                 log.info(f"Storage class {sc_name} already exists")
+                continue
+            pool_ocp = ocp.OCP(
+                kind=constants.CEPHBLOCKPOOL,
+                namespace=config.ENV_DATA["cluster_namespace"],
+                resource_name=pool_name,
+            )
+            if pool_ocp.is_exist(resource_name=pool_name):
+                log.info(f"Pool {pool_name} already exists, skipping creation")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)